functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -116,26 +118,38 @@ class JobUtils {
         return jobs[0]
     }
 
+
     static Execution waitForExecutionToBe(
             String state,
             String executionId,
             ObjectMapper mapper,
             RdClient client,
-            int iterationGap,
-            int timeout
+            WaitingTime iterationGap,
+            WaitingTime timeout
+    ) {
+        return waitForExecutionToBe(state, executionId, mapper, client, iterationGap.duration, timeout.duration)
+    }
+
+    static Execution waitForExecutionToBe(
```

Comment:
Looks like we have a duplicate method in BaseContainer that does very similar thing.
I refactored it yesterday, not being aware that  you're doing this:
https://github.com/rundeck/rundeck/pull/9283/files#diff-389e36f39f7e4b74b8e3e0c5e9996dafa659960d77d217cd06626c4bbb9bdaf4R371

Reply:
What i see is you're adding a similar method in BaseContainer in the other PR, i think you should use the method in JobUtils which is the one being used by all the other tests.

This method already existed, i'm just refactoring only for the purpose of having time units more explicit, and avoid all those `/ 1000 as int` in the tests.